spinta/formats/rdf/components.py

```diff
@@ -0,0 +1,31 @@
+import dataclasses
+from typing import Optional, Dict, Any
+
+from spinta.formats.components import Format
+
+
+class Rdf(Format):
+    content_type = 'application/rdf+xml'
+    accept_types = {
+        'application/rdf+xml',
+    }
+    params = {}
+
+
+@dataclasses.dataclass
+class Cell:
```

Comment:
I would change this name to `Elem`, or something like this, because RDF does not have things like cells, while HTML tables do have cells. Maybe instead of having a custom class, we could just use [etree.Element](https://docs.python.org/3/library/xml.etree.elementtree.html#xml.etree.ElementTree.Element)?